discussion/src/main/java/org/openedx/discussion/presentation/comments/DiscussionCommentsFragment.kt

```diff
@@ -310,7 +315,16 @@ private fun DiscussionCommentsScreen(
                 )
             )
         }
-
+        if (!uiState.equals(DiscussionResponsesUIState.Loading) && isloading){
```

Comment:
Logical issue: we’re comparing a `DiscussionCommentsUIState` to a `DiscussionResponsesUIState` value. Also, we don’t need this check here; `isLoading` is sufficient to control this UI.